pkg/tests/tasks/traffic/egress/egress_gateways_tls_file_mount_test.go

```diff
@@ -15,173 +15,198 @@
-		defer util.RecoverPanic(t)
-
-		log.Log.Info("Deploy nginx mtls server")
-		nginx := examples.Nginx{Namespace: "mesh-external"}
-		nginx.Install_mTLS(env.GetRootDir() + "/testdata/examples/x86/nginx/nginx_mesh_external_ssl.conf")
-
-		log.Log.Info("Redeploy the egress gateway with the client certs")
-		util.Shell(`kubectl create -n %s secret tls nginx-client-certs --key %s --cert %s`, meshNamespace, nginxClientCertKey, nginxClientCert)
-		util.Shell(`kubectl create -n %s secret generic nginx-ca-certs --from-file=%s`, meshNamespace, nginxServerCACert)
-
-		log.Log.Info("Patch egress gateway")
-		util.Shell(`kubectl -n %s rollout history deploy istio-egressgateway`, meshNamespace)
-		util.Shell(`kubectl -n %s patch --type=json deploy istio-egressgateway -p='%s'`, meshNamespace, strings.ReplaceAll(gatewayPatchAdd, "\n", ""))
-		time.Sleep(time.Duration(20) * time.Second)
-		util.Shell(`oc wait --for condition=Ready -n %s smcp/%s --timeout 180s`, meshNamespace, smcpName)
-		log.Log.Info("Verify the istio-egressgateway pod")
-		util.Shell(`kubectl exec -n %s "$(kubectl -n %s get pods -l %s -o jsonpath='{.items[0].metadata.name}')" -- ls -al %s %s`,
-			meshNamespace, meshNamespace,
-			"istio=egressgateway",
-			"/etc/istio/nginx-client-certs",
-			"/etc/istio/nginx-ca-certs")
-		util.Shell(`kubectl -n %s rollout history deploy istio-egressgateway`, meshNamespace)
-
-		log.Log.Info("Configure MTLS origination for egress traffic")
-		util.KubeApplyContents("bookinfo", util.RunTemplate(nginxGatewayTLSTemplate, smcp))
-		util.KubeApplyContents(meshNamespace, meshExternalServiceEntry)
-		util.KubeApplyContents(meshNamespace, nginxMeshRule)
-		time.Sleep(time.Duration(10) * time.Second)
-
-		log.Log.Info("Verify NGINX server")
-		cmd := fmt.Sprintf(`curl -sS http://my-nginx.mesh-external.svc.cluster.local`)
-		msg, err := util.PodExec("bookinfo", sleepPod, "sleep", cmd, true)
-		util.Inspect(err, "failed to get response", "", t)
-		if !strings.Contains(msg, "Welcome to nginx") {
-			t.Errorf("Expected Welcome to nginx; Got unexpected response: %s", msg)
-			log.Log.Errorf("Expected Welcome to nginx; Got unexpected response: %s", msg)
-		} else {
-			log.Log.Infof("Success. Get expected response: %s", msg)
-		}
 	})
 }
+
+var (
+	gatewayPatchAdd = `[{"op": "add","path": "/spec/template/spec/containers/0/volumeMounts/0","value": {"mountPath": "/etc/istio/nginx-client-certs","name": "nginx-client-certs","readOnly": true}},{"op": "add","path": "/spec/template/spec/volumes/0","value": {"name": "nginx-client-certs","secret": {"secretName": "nginx-client-certs","optional": true}}},{"op": "add","path": "/spec/template/spec/containers/0/volumeMounts/1","value": {"mountPath": "/etc/istio/nginx-ca-certs","name": "nginx-ca-certs","readOnly": true}},{"op": "add","path": "/spec/template/spec/volumes/1","value": {"name": "nginx-ca-certs","secret": {"secretName": "nginx-ca-certs","optional": true}}}]`
```

Comment:
You can convert this into a multi-line patch, just like Yann did here: https://github.com/maistra/maistra-test-tool/pull/432/files#diff-4a3df446451f30c96000873f8db9f5fc79c37d5d6927b9b4442f959768d5a342R76-R82

This makes the patch much easier to read and understand.

Oh, I see that it was a multi-line string before, so use that (but please add proper indentation to it)